From b17758992f8713417be05c21855e00d9756d81f6 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Fri, 1 Mar 2019 18:02:17 +0100
Subject: [PATCH 2/8] Revert "Use proper summary for network select list
 preference on dsds/dsda/tsts"

This reverts commit cbf6073ea51e07bc26c78312b02b37e47fa821b6.

Change-Id: Ia3d915081bcc444a864baa741a9e20d86aee6e73
---
 .../android/phone/NetworkSelectListPreference.java    | 11 +++++------
 1 file changed, 5 insertions(+), 6 deletions(-)

diff --git a/src/com/android/phone/NetworkSelectListPreference.java b/src/com/android/phone/NetworkSelectListPreference.java
index 6066abe99..341ac8d2b 100644
--- a/src/com/android/phone/NetworkSelectListPreference.java
+++ b/src/com/android/phone/NetworkSelectListPreference.java
@@ -318,16 +318,15 @@ public class NetworkSelectListPreference extends ListPreference
         mProgressDialog = progressDialog;
         mNeedScanAgain = false;
 
-        TelephonyManager telephonyManager = (TelephonyManager)
-                getContext().getSystemService(Context.TELEPHONY_SERVICE);
-
         if (SubscriptionManager.isValidSubscriptionId(mSubId)) {
             mPhoneId = SubscriptionManager.getPhoneId(mSubId);
-            setSummary(telephonyManager.getNetworkOperatorName(mSubId));
-        } else {
-            setSummary(telephonyManager.getNetworkOperatorName());
         }
 
+        TelephonyManager telephonyManager = (TelephonyManager)
+                getContext().getSystemService(Context.TELEPHONY_SERVICE);
+
+        setSummary(telephonyManager.getNetworkOperatorName());
+
         setOnPreferenceChangeListener(this);
     }
 
-- 
2.17.1

